TWLight/users/templates/users/filter_section.html

```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+<nav class="navbar navbar-expand-lg navbar-light">
+  <button class="navbar-toggler" type="button" data-toggle="collapse"
+      data-target="#filterNavbar" aria-controls="filterNavbar" aria-expanded="false"
+      aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div id="filterNavbar" class="collapse navbar-collapse filter-container">
+    <div class="row filter-title-container">
+      <p class="filter-title">
+        {% comment %}Translators: On the MyLibrary page (https://wikipedialibrary.wmflabs.org/users/my_library), this label indicates the start of the filter section. {% endcomment %}
+        {% trans "Filters: " %}
```

Comment:
I think we should not be including the `: ` in our strings for translation. We had a couple of these sneak in over the last pr or two in `available_collection_tile.html` and `user_collection_tile.html` and we should bring these all into line with what we're doing elsewhere, which is like `{% trans "Filters" %}: `

Reply:
Done!